### DISCUSSION TITLE
[Feature] Metadata editing

### DISCUSSION THREAD

**Discussion by djjudas21** [immich-app/immich#4938] [Feature Request]
### The feature

I'm a PhotoPrism user, but I like the look of Immich and I'm considering switching. There are a couple of killer features that I would like to see in Immich.

Sometimes I import images into my photo library which do not have EXIF metadata, for example:
- if I save them from WhatsApp or other social media
- if they were taken on early digital cameras that did not record EXIF (looking at you, Nokia 6230)
- if they were scanned from film negatives

PhotoPrism allows me to add/edit metadata to these images, so I can set the correct time, location, etc. I would love to see the ability to edit metadata in Immich.

![Screenshot from 2023-11-09 22-32-04](https://github.com/immich-app/immich/assets/604595/958382fc-1ba0-4ddc-928e-317ddc626ec3)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by thariq-shanavas** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/1696
